src/bitmessagekivy/tests/telenium_process.py

```diff
@@ -49,6 +50,18 @@ def populate_test_data():
                 os.path.abspath(os.path.dirname(__file__)), 'sampleData', file_name)
             new_destination_file = os.path.join(os.environ['BITMESSAGE_HOME'], file_name)
             shutil.copyfile(old_source_file, new_destination_file)
+        TeleniumTestProcess.reload_test_data()
+
+    @staticmethod
+    def reload_test_data():
+        """Reloading the test data"""
+        config.read(os.path.join(os.environ['BITMESSAGE_HOME'], 'keys.dat'))
```

Comment:
ok we discussed this, remember the analogy with the web server and a web browser? There are two processes, communicating through an API. The config should be read by the **server** (`kivy_main.py`), not by the **client** (`tests-kivy.py`).